app/lib/controllers/chat_list_controller.dart

```diff
@@ -113,13 +141,29 @@ class ChatListController extends GetxController {
       );
       newItems.add(newItem);
     }
-    roomItems = newItems;
+    joinedRooms = newItems;
     update(['chatlist']);
   }
 
   void moveItem(int from, int to) {
-    RoomItem item = roomItems.removeAt(from);
-    roomItems.insert(to, item);
+    JoinedRoom item = joinedRooms.removeAt(from);
+    joinedRooms.insert(to, item);
+    update(['chatlist']);
+  }
+
+  Future<void> acceptInvitation(String roomId) async {
+    await client.acceptInvitation(roomId);
+    // remove item from list
+    int index = invitationEvents.indexWhere((x) => x.roomId() == roomId);
+    invitationEvents.removeAt(index);
+    update(['chatlist']);
```

Comment:
Imo this should notify the ```GetBuilder``` to update the state.

Reply:
As you see, `id: 'chatlist'` contains `InviteInfoWidget`.
![image](https://user-images.githubusercontent.com/48786056/194399861-f0a24246-99ae-47f7-9d36-c4f78db1435f.png)

Reply:
I think you need to assign ```id: 'chatlist'``` to ```GetBuilder``` of ```buildListHeader``` to make it work.

Reply:
Although their state change scenarios are different. You can assign new id to these ```GetBuilder``` which are handling invitations instead of  ```chatlist``` key. And use ```update['new key']``` .